capstone-frontend/src/components/Login.jsx

```diff
@@ -21,18 +22,24 @@ export class Login extends Component {
   }
 
   loginResponseSuccess = (response) => {
-    this.setState({ isLogin: true });
-    this.setState({ profileObj: response.profileObj });
+    this.setState({ isLogin: true, googleUser: response, profileObj: response.profileObj });
```

Comment:
Do we need to store the profileObj as a separate state variable if it is already contained within the response? Couldn't we just access it via this.state.response.profileObj?

Reply:
I did try to remove the state profileObj, yet for some reason I am unable to access the contents of ```this.state.googleUser.profileObj```, so I left the state profileObj. This can be refactor in the future.